src/views/v2/components/read-more.php

```diff
@@ -9,15 +9,14 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.10
+ * @version TBD
  *
  */
 ?>
+<span class="tribe-events-c-small-cta-hellip"> &hellip; </span>
 <div class="tribe-events-c-small-cta tribe-common-b3 tribe-events-c-small-cta--readmore">
 	<a
 		href="<?php echo esc_url( get_permalink( get_the_ID() ) ) ?>"
 		class="tribe-events-c-small-cta__link tribe-common-cta tribe-common-cta--thin-alt"
-	>
-		<?php esc_html_e( 'Continue Reading' , 'the-events-calendar' ); ?>
-	</a>
+	><?php esc_html_e( 'Continue Reading' , 'the-events-calendar' ); ?></a>
```

Comment:
:no_entry_sign: Space found before comma in function call

<sub>hash: 47f74ca234a723b96dfbb2599797a970</sub>